### ISSUE TITLE
Junit report : traceback not encoded properly in utf-8

### ISSUE THREAD

**Issue by arsenik** [behave/behave#351]
I run my tests with:
```
behave --junit --junit-directory c:/tests/shared/junit
```

If an exception is thrown by a script, behave catch it and write it in junit xml report. But it is not encoded in UTF8 properly.  Looks at the Traceback part : 

```
Traceback (most recent call last):
  File "C:\Python27\lib\site-packagesehave\model.py", line 1456, in run
    match.run(runner.context)
  File "C:\Python27\lib\site-packagesehave\model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "C:	ests\sharedddeatures\stepsrowser.py", line 78, in visit_page
    open_browser(context)
```

**Comment by mixxorz** [comment]
Might be a duplicate of #336

**Comment by arsenik** [comment]
Yes it is exactly what #336 describes

**Comment by jenisys** [comment]
FIX for #336 was integrated into behave-1.2.6.dev0
